examples/gateway/cohere/cohere_example.py

```diff
@@ -0,0 +1,28 @@
+from mlflow.gateway import query, set_gateway_uri
+
+
+def main():
+    gateway_uri = "http://localhost:5000"
+
+    # Using the fluent API
+    set_gateway_uri(gateway_uri)
```

Comment:
```suggestion
    # Using the fluent API
    set_gateway_uri("http://localhost:5000")
```

Reply:
Can we update the comment? `Set the gateway URI` sounds more accurate.

Reply:
changed this in all 3 examples